tgui/packages/tgui/interfaces/AresInterface.jsx

```diff
@@ -365,6 +365,18 @@ const MainMenu = (props) => {
                 />
               </Stack.Item>
             )}
+            <Stack.Item>
+              <Button
+                content="Bioscan"
```

Comment:
Just a heads up that after https://github.com/cmss13-devs/cmss13/pull/6244 buttons should not use the content property. Instead their content needs to be as a child e.g.

```js
<Button content="Bioscan" />
```

Needs to instead be:
```js
<Button>Bioscan</Button>
```